infrastructure/dpladm/bin/dpladm-shared.source

```diff
@@ -111,6 +115,7 @@ EndOfMessage
     PRIMARY_DOMAIN="${singleRouteIndent}- \"${primaryDomain}\""
   fi
 
+
```

Comment:
Why this?

Reply:
to create some structure in the file, making it clear, that this is something new from the line above